### DISCUSSION TITLE
problem about building igl_copyleft_cgal.lib

### DISCUSSION THREAD

**Discussion by echo-hi** [libigl/libigl#2068] [Q&A]
I want to run the tutorial 609 ,but I failed to build igl_copyleft_cgal.lib after running the cmake of libigl.
here is the error to buld the  igl_copyleft_cgal.lib:
![image](https://user-images.githubusercontent.com/55911430/183818145-36228b00-6f7e-4d9d-89c2-a79967f5836f.png)

it could build the lib after i comment the code : 
   `auto* pet = new typename Lazy_exact_nt_rep<ET>::Indirect(cste);
    this->keep_at(pet);
    this->set_ptr(pet);`
but the lib would occur error when i test the tutorial 609

**Comment by echo-hi** [comment]
i would appreciate it if anyone answer this problem

**Reply by alecjacobson** [reply]
This seems like a CGAL bug/issue. Did you try asking on their github page?
